Refactor enterprise introduction and getting started (#1479)
emqtt/emq-docs-cn: en_US/getting-started/getting-started.md

Starting from download and installation, this document is to guide you to quickly get a EMQX node up and running.

{% emqxce %}

> ## Select EMQX Edition

Reviewer: ```suggestion
## Choose EMQX Software Edition or Service
```

EMQX as a software is released in different editions.

- [EMQX](https://www.emqx.com/en/try?product=broker)
- [EMQX Enterprise](https://www.emqx.com/en/try?product=enterprise)

EMQ as a company offers a fully managed MQTT cloud service [EMQX Cloud](https://www.emqx.com/en/try?product=cloud)
which runs EMQX Enterprise.
You can customize the deployment to suit your business development plans the best and get started quickly.

{% endemqxce %}

{% emqxee %}

> ## Deployment Mode

Reviewer: ```suggestion
## Choose Your Hosting Mode
```

> EMQX as a software provides the self-managed version [EMQX Enterprise](https://www.emqx.com/en/try?product=enterprise).

Reviewer: ```suggestion
EMQX Enterprise as a software provides a free-of-charge non-expiry trail license. You are welcome to [download it](https://www.emqx.com/en/try?product=enterprise).
```

### Run in a VM or on bare metal

EMQX can be deployed directly to bare metal servers or virtual machines.
A minimum of 2 cores and 4GB RAM can run the EMQX.

EMQX has prebuilt packages downloadable for Linux(RedHat, CentOS, Rockylinux)
as well as Debian, Ubuntu, and MacOS.

{% emqxce %}

- [CentOS installation](./install.md#centos)
- [Ubuntu, Debian installation](./install.md#ubuntu-debian)
- [MacOS, Windows, Linux ZIP installation](./install.md#zip-linux-macos-windows)

For other platforms which are not in the prebuilt list,
you may [build the runnables from source code](./install.md#source-code-compilation-and-installation).

{% endemqxce %}

{% emqxee %}

For other platforms which are not in the prebuilt list,
you may simply [contact EMQ](https://www.emqx.com/en/contact) for support.
{% endemqxee %}

> For other platforms which are not in the prebuilt list,
> you may {% emqxce %}[build the runnables from source code](./install.md#source-code-compilation-and-installation) or{% endemqxce %}
> simply [contact EMQ](https://www.emqx.com/en/contact) for support.

Reviewer: ```suggestion
{% emqxce %}
For other platforms which are not in the prebuilt list,
you may [build the runnables from source code](./install.md#source-code-compilation-and-installation) or
simply [contact EMQ](https://www.emqx.com/en/contact) for support.
{% endemqxce %}
```
